AutoDarkModeSvc/AutoDarkModeSvc.csproj

```diff
@@ -19,6 +19,10 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <SupportedOSPlatformVersion>10.0.18362.0</SupportedOSPlatformVersion>
+
+    <RuntimeIdentifiers>win10-x64;win10-arm64</RuntimeIdentifiers>
+    <WindowsPackageType>None</WindowsPackageType>
+    <WindowsAppSDKSelfContained>true</WindowsAppSDKSelfContained>
```

Comment:
Well i added that here but I'm not shure, because i tried without the `RuntimeIdentifiers``and it compile perfectly fine.

Reply:
ok my bad, i don't know why it remove the package